spec/features/web_intake/triage_spec.rb

```diff
@@ -138,7 +138,7 @@
         TaxReturn.current_tax_year - 1,
       ],
       income_type_options: ['none_of_the_above'],
-      assistance_options: ['chat'],
```

Comment:
It's simpler to select only one option here, e.g. `phone_review_english` (which is probably the common case). Not a blocker.